# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-pseudo/selection-contenteditable-011.html.ini
# Commit: 9b15baab6f77
# Full Hash: 9b15baab6f77d09eca2956276f66fee97555e4a1
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-09-22 09:56:47
# Regressor Bug: 1789967
# File Overlap Count: 1
# Description:
#   Bug 1789967 - part 3: Make `HTMLEditor::CollapseSelectionToEndOfLastLeafNodeOfDocument` and `HTMLEditor::InitEditorContentAndSelection` do nothing if the document is partially editable r=m_kato
#   
#   They and their callees work with the result of `GetRoot()` which is the document
#   element or the body element.  If the body is not editable, `Selection` should
#   not be updated in non-editable region nor `<br>` elements should not be
# ==============================================================================

diff -r 4d1e93c629da -r 9b15baab6f77 testing/web-platform/meta/css/css-pseudo/selection-contenteditable-011.html.ini
--- a/testing/web-platform/meta/css/css-pseudo/selection-contenteditable-011.html.ini	Thu Sep 22 06:24:38 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,2 +0,0 @@
-[selection-contenteditable-011.html]
-  expected: FAIL